src/tools/segmentation/FreehandScissorsTool.js

```diff
@@ -25,7 +25,7 @@ export default class FreehandScissorsTool extends BaseTool {
     const defaultProps = {
       name: 'FreehandScissors',
       strategies: {
-        FILL_INSIDE: fillInsideFreehand,
+        FILL_INSIDE: fillInsideFreehandScissors,
```

Comment:
Do you anticipate using the other strategies? If so, consider generalising your new function `fillInsideFreehandScissors` to take a callback as the original strategy function so that they all fire the event without having to add 3 more functions. Let me know if i can clarify

Reply:
942d832b927d0f4e9d13388d9b22b0d95829e1f9